src/main/java/database/helper/Settings.java

```diff
@@ -19,6 +19,11 @@ public class Settings {
     private static final Logger LOG = LoggerFactory.getLogger(Settings.class);
     static final String DATABASE_STORAGE_PATH = "database.storage.path";
     static final String DEFAULT_DATABASE_STORAGE_PATH = "database";
+    public static final String LIMIT = "limit";
```

Comment:
`LIMIT` constant can be unclear and can be recognized as a limit value. Consider about renaming parameter names to something like `LIMIT_PARAMETER_NAME`.